agent/whois_domain_agent.py

```diff
@@ -48,25 +48,31 @@ def process(self, message: msg.Message) -> None:
             message:  The message to process from ostorlab runtime.
         """
         domain = message.data.get("name")
-        if domain is None:
+        if domain is None or domain == "":
             return
 
         domain_object = cast(
-            tld.Result, tld.get_tld(domain, as_object=True, fix_protocol=True)
+            tld.Result,
+            tld.get_tld(domain, as_object=True, fix_protocol=True, fail_silently=True),
```

Comment:
why fail_silently=True ?

Reply:
for stop raising an error when the agent processing the message, and fail_silently=True, returns None in case of failing, and I already check that after getting tld.